lib/makeRequest.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const utils = require('./utils');
+const cloneDeep = require('lodash.clonedeep');
```

Comment:
I'd try to avoid the extra dependency if at all possible. Do you think it is necessary to deep clone? I haven't checked, but my guess would be we aren't doing any mutations more than a single level down and a shallow clone `{...opts}` would be enough.

Reply:
Using shallow copying with the `...` operator, the opts do get mutated, so I think we do need a deep copy. What works is `JSON.parse(JSON.stringify(args))`, of course this isn't the most robust way of copying. Any thoughts?

Reply:
Actually I may have misread your comment, I will try the shallow copy one level down and report back.

Reply:
Shallow copying one level down works! We do not need this dependency anymore. ptal @richardm-stripe